src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -718,7 +652,7 @@ impl<S: S3BackEnd + Sync + Send + 'static> MetaData for S3MetaData<S> {
 
             let child_ino = child_entry.ino();
             let child_node = self.get_inode_from_txn(txn.as_mut(), child_ino).await?;
-            let child_attr = self.apply_local_mtime_and_size(child_node.lookup_attr());
+            let child_attr = child_node.get_attr();
```

Comment:
Shall we look up to `open_files` for the attributes when the file is opened?

Reply:
No